libraries/lib-numeric-formats/tests/NumericConverterTests.cpp

```diff
@@ -213,7 +213,7 @@ TEST_CASE("BeatsNumericConverterFormatter", "")
             const auto value = bar * timeSignature.GetBarDuration() +
                                beat * timeSignature.GetBeatDuration();
 
-            basicFormatter->UpdateFormatForValue(value);
+            basicFormatter->UpdateFormatForValue(value, false);
```

Comment:
So it never shriks? I mean all calls to that function pass `false`

Reply:
Never indeed